### PR TITLE
Builds should fail if Turbine unit tests fail

### PR DESCRIPTION
**Pull request by rajivnathan**
Fixes https://github.com/eclipse/codewind/issues/916
The Jenkins PR builds should fail if there's a unit test failure.

Signed-off-by: Rajiv Senthilnathan <rajivsen@ca.ibm.com>

### CODE REVIEW

File: src/pfe/file-watcher/server/package.json [hunk 1]
```diff
@@ -14,7 +14,7 @@
     "test": "npm run unit:test && npm run functional:test",
     "unit:setup": "./test/scripts/unit_setup.sh",
     "unit:cleanup": "./test/scripts/unit_cleanup.sh",
-    "unit:test": "npm run unit:setup ; NODE_ENV=test HIDE_PFE_LOG=y CW_LOCALES_DIR=$(pwd)/src/utils/locales/ CW_WORKSPACE=$(pwd)/test/resources CW_PROJECTDATA_DIR=$(pwd)/test/resources/ nyc mocha test/unit-test/unit.test.ts ; npm run unit:cleanup",
+    "unit:test": "npm run unit:setup && NODE_ENV=test HIDE_PFE_LOG=y CW_LOCALES_DIR=$(pwd)/src/utils/locales/ CW_WORKSPACE=$(pwd)/test/resources CW_PROJECTDATA_DIR=$(pwd)/test/resources/ nyc mocha test/unit-test/unit.test.ts && npm run unit:cleanup",
     "unit:test:xml": "npm run unit:setup ; NODE_ENV=test HIDE_PFE_LOG=y CW_LOCALES_DIR=$(pwd)/src/utils/locales/  CW_WORKSPACE=$(pwd)/test/resources CW_PROJECTDATA_DIR=$(pwd)/test/resources/ nyc mocha test/unit-test/unit.test.ts --colors --reporter mocha-jenkins-reporter ; npm run unit:cleanup",
```
Comment (ssh24) [inline src/pfe/file-watcher/server/package.json]: @rajivnathan Could you also apply the same logic for the xml part?

Comment (rajivnathan) [reply src/pfe/file-watcher/server/package.json]: Done

**Comment by ssh24** [comment]
Nice! PR builds are failing as expected. Looks like https://github.com/eclipse/codewind/pull/921 landed, so if you rebase and try again it should work now.

**Comment by rajivnathan** [comment]
Verified that test failures now fail the build.

```
08:19:55  npm ERR! code ELIFECYCLE
08:19:55  npm ERR! errno 1
08:19:55  npm ERR! file-watcher@1.0.1 unit:test: `npm run unit:setup && NODE_ENV=test HIDE_PFE_LOG=y CW_LOCALES_DIR=$(pwd)/src/utils/locales/ CW_WORKSPACE=$(pwd)/test/resources CW_PROJECTDATA_DIR=$(pwd)/test/resources/ nyc mocha test/unit-test/unit.test.ts && npm run unit:cleanup`
08:19:55  npm ERR! Exit status 1
08:19:55  npm ERR! 
08:19:55  npm ERR! Failed at the file-watcher@1.0.1 unit:test script.
08:19:55  npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
08:19:55  
08:19:55  npm ERR! A complete log of this run can be found in:
08:19:55  npm ERR!     /home/genie.codewind/.npm/_logs/2019-10-31T12_19_55_078Z-debug.log
08:19:55  +++   TURBINE UNIT TESTS FAILED   +++
```

Will now rebase on master which has the fixes and then merge.

**Pull request merged**